src/hooks/vaultsHooks.ts

```diff
@@ -214,20 +103,31 @@ const useDebouncedMarketsData = () => {
   // wrap in object because the stupud abacus value isn't a new reference when data is new for some reason
   const [marketsToReturn, setMarketsToReturn] = useState<{
     data: Nullable<kollections.Map<string, PerpetualMarket>>;
-  }>({ data: latestMarkets.current });
+  }>({
+    data:
+      latestMarkets.current != null && latestMarkets.current.size > 0
```

Comment:
does this mean 
if `latestMarkets.current != null && (latestMarkets.current.size > 0 ? latestMarkets.current : undefined)`
or 
`(latestMarkets.current != null && latestMarkets.current.size > 0) ? latestMarkets.current : undefined`

i think functionally does the same thing here for now, but some parentheses would go a long way here esp if we need to update this in the future

Reply:
Rewrote this with jared's suggestions. In this case the linter won't let us add clarifying parentheses iirc